themes/classic/_dev/js/customer.js

```diff
@@ -0,0 +1,18 @@
+import $ from 'jquery';
+
+function initRmaItemSelector () {
+  $('#order-return-form table thead input[type=checkbox]').on('click', function () {
+    var checked = $(this).prop('checked');
+    $('#order-return-form table tbody input[type=checkbox]').each(function (_, checkbox) {
+      $(checkbox).prop('checked', checked);
+    })
```

Comment:
Please add `;` at the end of the calls, if someone add something later and forget to add it, a minified version can crash.